utils.sh

```diff
@@ -22,6 +22,54 @@ done
 sudo cp "$current" "$target"
 }
 
+function patch_node_ignition() {
+    local kind
+    local bootstrap
+    local wd
+    local config
+
+    kind="$1"
+    bootstrap="$2"
+    wd="$(mktemp -d)"
+    ssh_opts=("-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null")
```

Comment:
nit: do we need parenthesis here?

Reply:
I'm making it into an array. That's why I put the parenthesis there. This way I can properly quote the var when I use it.